docs/products/kafka/concepts/upgrade-procedure.rst

```diff
@@ -0,0 +1,253 @@
+Apache Kafka® upgrade procedure
+===============================
+
+One of the benefits of using a managed service like Aiven for Apache Kafka® is the automated upgrade procedure. 
+
+The upgrade procedure is executed during:
+    
+* maintenance updates
+* plan changes
+* cloud region migrations
+* manual node replacements performed by an Aiven operator
+    
+All the above operations involve creating new broker nodes to replace existing ones.
+
+Upgrade procedure steps
+---------------------------
+
+The following set of steps are executed during an upgrade procedure, the example shows a possible scenario for a 3-node Apache Kafka service.
+
```

Comment:
Actually, for the readability sake maybe better to do it a bit different (see if you like this idea or dismiss it):

  "To demonstrate what steps are taken during the automated upgrade procedure, we'll look at the example for a 3-node Apache Kafka service visualised below:"

Then the initial visualisation follows

"The following set of steps are executed during an upgrade procedure:"
1....
2....
3...